package.json

```diff
@@ -18,12 +18,14 @@
   ],
   "main": "./lib/index.js",
   "scripts": {
+    "gulp": "gulp",
     "build": "run-p --aggregate-output build:*",
     "build-watch": "run-p -l build-watch:*",
     "build-watch:client": "cross-env NODE_ENV=development webpack -w",
     "build-watch:server": "cross-env NODE_ENV=development babel -w ./src/server -d ./lib",
     "build:client": "cross-env NODE_ENV=production webpack",
     "build:server": "cross-env NODE_ENV=production babel ./src/server -d ./lib",
+    "build:styles": "gulp",
```

Comment:
Do this need to be added with the more common use of `npx gulp` to run the module from inside the project?